requirements.txt

```
@@ -19,7 +19,7 @@ beautifulsoup4>=4,<=4.6.3
 pyelftools>=0.24,<=0.25
 manifest-tool==1.5.2
 icetea>=1.2.1,<1.3
-pycryptodome>=3.7.2,<=3.7.3
+pycryptodome==3.9.3
```

Suggest using >=3.9.3,<4 here, rather than pinning the version
```suggestion
pycryptodome>=3.9.3,<4
```

---

Agreed. Fixed.